c/src/error.rs

```diff
@@ -30,6 +31,14 @@ thread_local! {
     static LAST_ERROR: RefCell<Option<Error>> = RefCell::new(None);
 }
 
+#[no_mangle]
+pub extern "C" fn init_logging() {
+    const ENV_VAR: &str = "TYPEDB_DRIVER_LOG_LEVEL";
+    if let Err(err) = env_logger::try_init_from_env(Env::new().filter(ENV_VAR)) {
+        warn!("{err}");
+    }
+}
```

Comment:
This function allows the FFI drivers to initialize the Rust logger. (Rust driver users can do this directly)